JavaScript and Ajax/ClassDetailPage.page

```diff
@@ -0,0 +1,38 @@
+<apex:page standardController="Class__c">
```

Comment:
page name is in correct this page is used to create/add a student to the class

Reply:
changed it.